lib/app/components/pages/dashboard/views/dashboard_view.dart

```diff
@@ -5,21 +5,39 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
 import '../controllers/dashboard_controller.dart';
+import 'package:google_mobile_ads/google_mobile_ads.dart';
 
 class DashboardView extends GetView<DashboardController> {
   const DashboardView({Key? key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Obx(
-        () => IndexedStack(
-          index: controller.tabService.tabIndex.value,
-          children: const [
-            TabRootView(initialRoot: Routes.home),
-            TabRootView(initialRoot: Routes.chat),
-            TabRootView(initialRoot: Routes.mypage)
-          ],
-        ),
+      body: Column(
+        children: [
+          Flexible(
+            child: Obx(
+              () => IndexedStack(
+                index: controller.tabService.tabIndex.value,
+                children: const [
+                  TabRootView(initialRoot: Routes.home),
+                  TabRootView(initialRoot: Routes.chat),
+                  TabRootView(initialRoot: Routes.mypage)
+                ],
+              ),
+            ),
+          ),
+          Obx(() => Container(
+                width: controller.isShowAd.value
+                    ? controller.getAd()!.size.width.toDouble()
+                    : 0,
+                height: controller.isShowAd.value ? 72.0 : 0,
+                alignment: Alignment.center,
+                child: controller.isShowAd.value
+                    ? AdWidget(ad: controller.getAd()!)
+                    : const Spacer(),
+              ))
```

Comment:
```suggestion
          Obx(
            () => Visibility(
              visible: controller.isShowAd.value,
              child: Container(
                width: controller.getAd()?.size.width.toDouble(),
                height: 72.0,
                alignment: Alignment.center,
                child: controller.isShowAd.value
                    ? AdWidget(ad: controller.getAd()!)
                    : const Spacer(),
              ),
            ),
          )
```
表示/非表示の処理はVisibilityでもいい気がするのですがどうですか？
childのWidgetは三項演算子抜けないですが、考えることが減るのでこっちの方がいい気がします。

Reply:
ありがとうございます！Visibilityを利用する形で良いと思います。修正いたしました。